### ISSUE TITLE
Plugin not compatible with Camunda Modeler 3.7.0 (macOs)?

### ISSUE THREAD

**Issue by stefanzilske** [mesoneer-ag/camunda-modeler-property-info-plugin#22]
When I use the plugin with Modeler 3.7.0 (on macOS 10.14.6) and enable Overlays on a fairly simple process (without listeners etc), overlays won't show, but an error is thrown instead:

```
Cannot read property 'extensionElements' of undefined
    at addStyle (file:///Users/zilskes/Library/Application%20Support/camunda-modeler/plugins/camunda-modeler-property-info-plugin/client/client-bundle.js:124:71)
    at toggleOverlays (file:///Users/zilskes/Library/Application%20Support/camunda-modeler/plugins/camunda-modeler-property-info-plugin/client/client-bundle.js:73:21)
    at Object.togglePropertyOverlays (file:///Users/zilskes/Library/Application%20Support/camunda-modeler/plugins/camunda-modeler-property-info-plugin/client/client-bundle.js:34:13)
    at opts (webpack:///node_modules/diagram-js/lib/features/editor-actions/EditorActions.js:170:31)
    at trigger (webpack:///src/app/tabs/bpmn/BpmnEditor.js:610:33)
    at triggerAction (webpack:///src/app/tabs/MultiSheetTab.js:258:18)
    at triggerAction (webpack:///src/app/tabs/EditorTab.js:36:33)
    at triggerAction (webpack:///src/app/App.js:1678:15)
    at fn (webpack:///src/app/App.js:2188:19)
    at file:///Applications/Camunda%20Modeler.app/Contents/Resources/app.asar/public/bundle.js:51:129365 [ error ]
This error may be the result of a plug-in compatibility issue. [ info ]
Disable plug-ins (restarts the app) [ info ]
```

Is there a compatibility issue with latest Modeler version?

**Comment by harrybraun** [comment]
I get the same error message on Windows 10 with Modeler 3.7.0.

Info: After an update to 3.7.1 everything runs perfectly again!

**Comment by dbudke** [comment]
Can you post the bpmn, please? Have you tried with Modeler 3.7.1 or 4.0.0 alpha1? 
![image](https://user-images.githubusercontent.com/15988866/78227037-f0c7d580-74cc-11ea-933b-2c3b230d65dd.png)
![image](https://user-images.githubusercontent.com/15988866/78227182-2f5d9000-74cd-11ea-891b-5a8ca84a767a.png)

**Comment by dbudke** [comment]
Issue solved in Camunda Modeler 3.7.1. Thx for your Feedback @harrybraun

**Comment by dbudke** [comment]
Issue solved in Camunda Modeler 3.7.1. Thx for your Feedback @harrybraun
